core/gen_genesis.go

```diff
@@ -48,7 +48,8 @@ func (g Genesis) MarshalJSON() ([]byte, error) {
 	if g.Alloc != nil {
 		enc.Alloc = make(map[common.UnprefixedAddress]GenesisAccount, len(g.Alloc))
 		for k, v := range g.Alloc {
-			enc.Alloc[common.UnprefixedAddress(k)] = v
+			internal, _ := k.InternalAddress()
```

Comment:
We should probably handle this error

Reply:
I don't think this file is used (`Genesis` is missing fields) but fixed